xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -36,7 +36,7 @@ extension AppTheme {
     static let cornerRadius: Double = 8
     static let cornerRadiusLg: Double = 16
     
-    static let smPfpSize: CGFloat = 24
+    static let smPfpSize: CGFloat = 22
```

Comment:
In Figma the backlinks panel seems to have 24pt pfps but the transcludes have 22pt pfps. I think 22 looks better for transcludes and I'm tempted to use that for both cases.